src/components/organisms/ToDoList/index.tsx

```
@@ -0,0 +1,120 @@
+import styled from '@emotion/styled'
+import { Box, Grid, Modal } from '@mui/material'
+import React, { useEffect, useState } from 'react'
+import MuiTypograpy from '../../atoms/MuiTypograpy'
+import MuiButton from '../../atoms/MuiButton'
+import MuiTextField from '../../atoms/MuiTextField'
+import { setSelectionRange } from '@testing-library/user-event/dist/utils'
+import { AddData, DeleteData, UpdateData, fetchtoDoData } from '../../../services'
+
+const OuterContainer=styled(Grid)({
+    height:'500px',
+    width:'1000px',
+    background:'lightgrey'
+})
+
+const SearchContainer=styled(Grid)({
+    display:'flex',
+    alignItems:'center',
+    justifyContent:"space-between"
+})
+const RenderContainer=styled(Grid)({
+    background:'black',
+    height:'auto',
+    padding:'20px'
+})
+const InsideContainer=styled(Grid)({   
+justifyContent:'space-between',
+display:'flex',
+marginBottom:'10px',
+background:'lightgreen'
+})
+const handleStyle = {
+    position: 'absolute' as 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    }
+    
+const TodoList = () => {
+    const [search,setSearchData]=useState("");
+    const [todoData,setTodoData]=useState([]);
+    const [open,setOpen]=useState<boolean>(false)
+
+    useEffect(()=>{
+          fetchtoDoData().then((response)=>{
+            console.log(response.data);
+            setTodoData(response.data);
+          }).catch((err)=>{
+           console.log(err);
+          })
```

nit : instead of logging error throw that error message